orc8r/cloud/go/services/orchestrator/obsidian/handlers/gateway_handlers.go

```diff
@@ -278,7 +299,14 @@ func LoadMagmadGateway(ctx context.Context, networkID string, gatewayID string)
 	if dev != nil {
 		devCasted = dev.(*models.GatewayDevice)
 	}
-	return (&models.MagmadGateway{}).FromBackendModels(ent, devCasted, status), nil
+
+	gateway := (&models.MagmadGateway{}).FromBackendModels(ent, devCasted, status)
+	gateway, err = models.PopulateRegistrationInfo(ctx, gateway, networkID)
+	if err != nil {
+		return nil, obsidian.MakeHTTPError(err, http.StatusInternalServerError)
+	}
+
+	return gateway, nil
```

Comment:
Add populating into gateway

```suggestion
	gateway := (&models.MagmadGateway{}).FromBackendModels(ent, devCasted, status)
	return gateway, nil
```